### PR TITLE
Remove warnings

### PR DESCRIPTION
**Pull request by fsx950223**

### CODE REVIEW

**Comment by mihaimaruseac** [comment]
Can you resolve conflicts please?

**Comment by fsx950223** [comment]
> Can you resolve conflicts please?

Resolved

File: tensorflow_estimator/contrib/estimator/python/estimator/rnn_test.py [hunk 1]
```diff
@@ -642,7 +642,7 @@ def _minimize(loss, global_step):
       if expected_loss is None:
         return state_ops.assign_add(global_step, 1).op
       assert_loss = _assert_close(
-          math_ops.to_float(expected_loss, name='expected'),
+          math_ops.cast(expected_loss, name='expected', dtype=dtypes.float32),
```
Comment (mihaimaruseac) [inline tensorflow_estimator/contrib/estimator/python/estimator/rnn_test.py]: Please also run pylint as this line seems too long

File: tensorflow_estimator/python/estimator/canned/baseline_estimator_test.py [hunk 2]
```diff
@@ -109,7 +109,7 @@ def get_updates(self, loss, params):
           return [self.iterations.assign_add(1).op]
         return [control_flow_ops.no_op()]
       assert_loss = assert_close(
-          math_ops.to_float(expected_loss, name='expected'),
+          math_ops.cast(expected_loss, name='expected', dtype=dtypes.float32),
```
Comment (mihaimaruseac) [inline tensorflow_estimator/python/estimator/canned/baseline_estimator_test.py]: This line is also long

File: tensorflow_estimator/python/estimator/canned/linear.py [hunk 3]
```diff
@@ -69,7 +69,6 @@ class LinearSDCA(object):
 
   SDCA can only be used with `LinearClassifier` and `LinearRegressor` under the
   following conditions:
-
```
Comment (mihaimaruseac) [inline tensorflow_estimator/python/estimator/canned/linear.py]: Don't remove this line please

**Comment by fsx950223** [comment]
> Please make sure to run pylint on the code

Fixed

File: tensorflow_estimator/python/estimator/canned/linear.py [hunk 4]
```diff
@@ -157,7 +157,8 @@ def _prune_and_unique_sparse_ids(self, id_weight_pair):
           [array_ops.shape(id_tensor.indices)[0]], dtypes.float32)
 
     example_ids = array_ops.reshape(id_tensor.indices[:, 0], [-1])
-    flat_ids = math_ops.to_int64(array_ops.reshape(id_tensor.values, [-1]))
+    flat_ids = math_ops.cast(
+        array_ops.reshape(id_tensor.values, [-1], dtype=dtypes.int64))
```
Comment (rthadur) [inline tensorflow_estimator/python/estimator/canned/linear.py]: Internal Test failures : `in _prune_and_unique_sparse_ids
    array_ops.reshape(id_tensor.values, [-1], dtype=dtypes.int64))
TypeError: reshape() got an unexpected keyword argument 'dtype'  `

Comment (mihaimaruseac) [reply tensorflow_estimator/python/estimator/canned/linear.py]: Missed this on review, but there should be a `)` immediately after `[-1]` and only one `)` after `int64`.

Thanks for catching it out. @fsx950223 can you do the change please?

Comment (fsx950223) [reply tensorflow_estimator/python/estimator/canned/linear.py]: Fixed

**Pull request merged**